pkg/operator/ceph/object/spec.go

```diff
@@ -426,7 +426,11 @@ func (c *clusterConfig) vaultPrefixRGW() string {
 		vaultPrefixPath = path.Join(vaultPrefixPath,
 			c.store.Spec.Security.KeyManagementService.ConnectionDetails[vault.VaultBackendPathKey], "/data")
 	case kms.VaultTransitSecretEngineKey:
-		vaultPrefixPath = path.Join(vaultPrefixPath, secretEngine, "/export/encryption-key")
+		if c.clusterInfo.CephVersion.IsAtLeastPacific() {
```

Comment:
How does this work on upgrades? Let's say someone deployed in Octopus had `/export/encryption-key` now the code will switch to `/transit`, won't we have an issue?

Reply:
Even I am not quite sure how it works internally, the upstream docs says the token need to access for both old and new policies, https://docs.ceph.com/en/latest/radosgw/vault/#transit-engine-compatibility-support. Nothing mentioned wrt backend paths
@mdw-at-linuxbox ??